.ci/pipeline/ci.yml

```diff
@@ -714,9 +679,13 @@ jobs:
       bash .ci/scripts/describe_system.sh
     displayName: 'System info'
   - script: |
+      call C:\temp\oneapi\setvars.bat --force
+      set MKL_FPK_GPU_VERSION_LINE=2024.0.0
```

Comment:
Why GPU FPK is still mentioned?

Reply:
Its necessary for choosing correct layout of oneTBB lib, we still have in makefile this flag. Flag decides is it lib/intel64 or lib/ layout